sproutset.py

```diff
@@ -0,0 +1,18 @@
+class Settings():
+    def __init__(self):
+
+            #Dimensions
+            self.width = 1000
+            self.height = 600
+
+            self.coordinates = 500, 25, 25
+
+            #Colours
+            self.RED = (255, 0, 0)
+            self.BLACK = (0, 0, 0)
+            self.BLUE = (0, 0, 255)
+            self.WHITE = (255, 255, 255)
+            self.ORANGE = (255,165,0)
+            self.GREEN = (0, 255, 0)
+
+            
```

Comment:
About version control and other stuff: looks very tidy! I think, however, that there are 1-2 things missing, for instance, the license agreement (maybe I am blind though) and the setup file... :)

Reply:
oh and don't forget to include a unit test when you are ready :)

Reply:
They were missing you're right. As I hadn't made them yet